### PR TITLE
OCPBUGS-33460: E2E: change tuned profile path 

### PR DESCRIPTION
**Pull request by mrniranjan**

### CODE REVIEW

**Review by MarSik — APPROVED** [review]
/lgtm

File: test/e2e/performanceprofile/functests/1_performance/netqueues.go [hunk 1]
```diff
@@ -137,7 +142,7 @@ var _ = Describe("[ref_id: 40307][pao]Resizing Network Queues", Ordered, func()
 			}
 			//Verify the tuned profile is created on the worker-cnf nodes:
 			tunedCmd := []string{"bash", "-c",
-				fmt.Sprintf("cat /etc/tuned/openshift-node-performance-%s/tuned.conf | grep devices_udev_regex", performanceProfileName)}
+				fmt.Sprintf("cat %s | grep devices_udev_regex", tunedConfPath)}
```
Comment (jmencak) [inline test/e2e/performanceprofile/functests/1_performance/netqueues.go:145]: Nit: I realize you didn't write that originally, but would it be better written without the `cat`?  `grep` can search in files too and you will not need a pipe and an extra process invocation.  Similarly elsewhere.
```

Comment (mrniranjan) [reply test/e2e/performanceprofile/functests/1_performance/netqueues.go:145]: ack fixing it

**Review by MarSik — APPROVED** [review]
/lgtm

**Pull request merged**